composer.json

```diff
@@ -97,6 +97,9 @@
         ]
     },
     "config": {
+		"platform": {
+			"php": "8.0.2"
```

Comment:
I am running php8.1 locally, if I do `composer install` or `composer update` it will lock with php8.1 compatibility.
Some of the updated packages are not php8.0 backward compatible, thus this requirement.